src/components/look-controls.js

```diff
@@ -28,7 +28,11 @@ module.exports.Component = registerComponent('look-controls', {
 
   play: function () {
     this.previousPosition.set(0, 0, 0);
-    this.addEventListeners();
+
+    // defer adding listeners until everything has run.
```

Comment:
`this.el.sceneEl.canvas` is undefined.    The scene canvas isn't setup before the event listener runs.

I tried using `loaded` event, but for some reason it runs multiple times on scene.  Not sure why this is.